newrelic.js

```diff
@@ -0,0 +1,11 @@
+const cfenv = require("cfenv")
+const appEnv = cfenv.getAppEnv()
+const newRelicEnv = appEnv.getServiceCreds("analytics-reporter-new-relic")
```

Comment:
After playing around with it, I'm going to stick with an env var. Switching up the way we manage variables like this may need to wait for a refactor where we can do it wholesale.